app/views/navbar.scala.html

```diff
@@ -179,69 +179,64 @@
         });
     }
 
-    // Triggered when 'Sign In' in navbar is clicked
-    // Logs "Click_module=SignIn_location=Navbar_route=</|/audit|/help|...>"
-    $("#sign-in-button").on('click', function(){
+    // The rest of this script logs whichever button is hit in the navbar
```

Comment:
As per [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide), add period to end of comment.